libraries/tock-tbf/src/types.rs

```diff
@@ -310,6 +371,43 @@ impl core::convert::TryFrom<&[u8]> for TbfHeaderV2Main {
     }
 }
 
+impl core::convert::TryFrom<&[u8]> for TbfHeaderV2Program {
+    type Error = TbfParseError;
+    fn try_from(b: &[u8]) -> Result<TbfHeaderV2Program, Self::Error> {
+        // For 3 or more fields, this shortcut check reduces code size
+        if b.len() < 20 {
+            return Err(TbfParseError::InternalError);
+        }
+        Ok(TbfHeaderV2Program {
+            init_fn_offset: u32::from_le_bytes(
+                b.get(4..8)
```

Comment:
The change to this line shouldn't have been in the commit.
```suggestion
                b.get(0..4)
```

Reply:
Yup -- just figured it out because processes weren't loading. Fixed it.